mappings/com/mojang/blaze3d/texture/NativeImage.mapping

```diff
@@ -76,6 +76,15 @@ CLASS net/minecraft/unmapped/C_ayikuhxa com/mojang/blaze3d/texture/NativeImage
 	METHOD m_guvwjisj getRed (II)B
 		ARG 1 x
 		ARG 2 y
+	METHOD m_hmadgtdg copyRect (IIIIIIZZ)V
+		ARG 1 srcX
+		ARG 2 srcY
+		ARG 3 shiftX
+		ARG 4 shiftY
```

Comment:
I think we use offset pretty much everywhere

Reply:
```suggestion
		ARG 3 offsetX
		ARG 4 offsetY
```